packages/pluggableWidgets/pie-doughnut-chart-native/src/__tests__/PieDoughnutChart.spec.tsx

```diff
@@ -0,0 +1,340 @@
+import { EditableValueBuilder, ListAttributeValueBuilder, ListValueBuilder } from "@mendix/piw-utils-internal";
+import { act, fireEvent, render, Thenable } from "react-native-testing-library";
+import { ReactTestInstance } from "react-test-renderer";
+import { ObjectItem } from "mendix";
+import { View } from "react-native";
+import { createElement } from "react";
+import { Big } from "big.js";
+
+import { PieDoughnutChartProps, SeriesType } from "../../typings/PieDoughnutChartProps";
+import { ChartStyle, defaultStyle } from "../ui/Styles";
+import { PieDoughnutChart } from "../PieDoughnutChart";
+
+jest.mock("victory-native", () => ({
+    VictoryPie: jest.fn(({ children, ...rest }) => (
+        <View testID="mock" {...rest}>
+            {children}
+        </View>
+    ))
+}));
+
+describe("PieDoughnutChart", () => {
+    describe("configuration", () => {
+        it("basic static props", () => {
+            const { getByTestId, toJSON } = render(<PieDoughnutChart {...createProps()} />);
+            fireLayoutEvent(getByTestId("innerView"));
+
+            expect(toJSON()).toMatchSnapshot();
+        });
+
+        it("renders doughnut correctly with default inner radius", () => {
+            const { getByTestId } = render(<PieDoughnutChart {...createProps({ presentation: "doughnut" })} />);
+            fireLayoutEvent(getByTestId("innerView"));
+
+            expect(getByTestId("mock").props.innerRadius).toEqual(25);
+        });
+
+        it("renders doughnut correctly with custom inner radius", () => {
+            const { getByTestId } = render(
+                <PieDoughnutChart
+                    {...createProps({
+                        presentation: "doughnut",
+                        style: [
+                            {
+                                slices: {
+                                    innerRadius: 111
+                                }
+                            }
+                        ]
+                    })}
+                />
+            );
+            fireLayoutEvent(getByTestId("innerView"));
+
+            expect(getByTestId("mock").props.innerRadius).toEqual(111);
+        });
+
+        it("handles labels", () => {
+            const { getByTestId } = render(<PieDoughnutChart {...createProps({ showLabels: true })} />);
+            fireLayoutEvent(getByTestId("innerView"));
+
+            expect(getByTestId("mock").props.labels({ datum: { x: "label1" } })).toEqual("label1");
+        });
+
+        it("handles sortOrder", () => {
+            const { getByTestId, toJSON } = render(<PieDoughnutChart {...createProps({ sortOrder: "descending" })} />);
+            fireLayoutEvent(getByTestId("innerView"));
+
+            expect(toJSON()).toMatchSnapshot();
```

Comment:
What are you testing with this snapshot? Because as far as I can see from the actual snapshot, there is no data there to verify that the sort order actually does something.